src/com/teamcautionrobotics/robot2018/Robot.java

```diff
@@ -87,6 +90,27 @@ public void teleopPeriodic() {
             spinSpeed = manipulator.getAxis(Axis.RIGHT_TRIGGER);
         }
         intake.spin(spinSpeed);
+
+        boolean liftRaiseButton = manipulator.getButton(Button.Y);
+        if (liftRaiseButton != liftRaiseButtonPressed) {
+            if (liftRaiseButton) {
+                lift.setLevel(lift.getCurrentLiftLevel().next());
+            }
+            liftRaiseButtonPressed = liftRaiseButton;
+        }
+
+        boolean liftLowerButton = manipulator.getButton(Button.Y);
+        if (liftLowerButton != liftRaiseButtonPressed) {
+            if (liftLowerButton) {
+                lift.setLevel(lift.getCurrentLiftLevel().previous());
+            }
+            liftLowerButtonPressed = liftLowerButton;
+        }
+
+        // manual control
+        double manualLiftAxis = manipulator.getAxis(Axis.RIGHT_Y);
+        double changeInHeight = LIFT_NUDGE_SPEED * manualLiftAxis * dt; // inches
+        lift.setHeight(lift.getCurrentHeight() + changeInHeight);
```

Comment:
The accumulator (`changeInHeight`) does not accumulate. It should be a field of Robot.

Reply:
I was wrong the height adjust actually is integrated by changing lift's height.